driver/src/main/com/mongodb/MongoClientOptions.java

```diff
@@ -346,7 +350,9 @@ public boolean isSocketKeepAlive() {
 
     /**
      * Gets the heartbeat frequency. This is the frequency that the driver will attempt to determine the current state of each server in the
-     * cluster. The default value is 10,000 milliseconds.
+     * cluster.
+     *
+     * <p>Default is 10,000.</p>
```

Comment:
It would be best to keep the timeunits for all the timeouts, so the user doesn't have to look it up. eg:

`<p>Default is 10,000.</p>`